community/contributing/docs/local-clone.md

```diff
@@ -0,0 +1,263 @@
+
+   community/contributing/docs/local-clone.md
+
+   no changes added to commit (use "git add" and/or "git commit -a")
+   ```
+
+1. Add new files listed under **Untracked files** to the commit:
+
+   ```bash
+   git add <your-file-name>
+   ```
+
+   Repeat this for each new file.
+
+1. Create a commit:
+
+   ```bash
+   git commit -a -m "<your-commit-subject>" -m "<your-commit-description>"
+   ```
+
+   - `-a`: Commits all changes.
+   - `-m`: Use the given `<msg>` as the commit message. If multiple `-m` options are given, their values are concatenated as separate paragraphs.
+
+   Your commit messages must be 50 characters or less. Do not use any [GitHub
+   Keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) in your commit message. You can add those to the pull request description later.
+
+1. Push your working branch and its new commit to your remote fork:
+
+   ```bash
+   git push origin <your-working-branch>
+   ```
+
+## Create a pull request from your fork to spinnaker/spinnaker.github.io
+
+1. In a web browser, go to the `spinnaker/spinnaker.github.io` [repository](https://github.com/spinnaker/spinnaker.github.io). You should see your recently pushed working branch with a **Compare & pull request** button.
+
+   ![CompareAndPullRequest](/assets/images/community/contributing/docs/compare-and-pr.jpg)
+
+1. Click **Compare & pull request**. This takes you to the **Open a pull request** screen.
+
+   ![OpenPullRequest](/assets/images/community/contributing/docs/github-open-pull-request.jpg)
+
+   1. The **Title** defaults to the commit subject. Update the title so it follows the `<type>(<scope>): <subject>` format. Make sure you include a space after the colon. For example:
+
+      ```
+      docs(plugins): add documentation for plugin creators
+      ```
+
+      The Spinnaker repositories use a PR title checker, so your PR will fail if the title is not in the correct format. For more information, see [commit message conventions](/community/contributing/submitting/#commit-message-conventions).
+
+   2. The **Leave a comment** field defaults to the commit description. Pull request descriptions are the first step to helping reviewers and project maintainers understand why your change was made. Do not leave this field blank. Provide as much description as possible. A good description helps get your PR merged faster!
+   3. Leave the **Allow edits from maintainers** checkbox selected.
+
+1. Click the **Create pull request** button.
+
+Congratulations! You can view your submitted pull request on the **Pull requests** [tab](https://github.com/spinnaker/spinnaker.github.io/pulls).
+
+>Do not delete your working branch until your pull request has been merged!
+
+## Addressing feedback locally
+
+Reviewers may ask you to make changes to your pull request. Read the feedback and make changes in your working branch.
+
```

Comment:
Do you want to suggest fetching/rebasing first, in case the repo's changed in the mean time?

Reply:
Oh, now I see you mention it below. But it might be good to put it here (too). Besides reviewers suggesting changes on your branch, stuff might have gotten merged into master under other PRs in the meantime.

Reply:
I don't usually recommend rebasing unless the GitHub UI indicates a merge conflict. It's too easy to end up in rebase hell. I added a few lines to clarify messages contributors might see.